stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/view/FootnoteView.kt

```diff
@@ -67,6 +69,52 @@ internal class FootnoteView : ConstraintLayout {
             threadRepliesButton.text =
                 resources.getQuantityString(R.plurals.stream_ui_thread_messages_indicator, replyCount, replyCount)
         }
+        setupUserAvatars(isMine, threadParticipants)
+    }
+
+    private fun setupUserAvatars(isMine: Boolean, threadParticipants: List<User>) {
+        fun applyUser(user: User?, avatarView: AvatarView) {
+            if (user != null) {
+                avatarView.setUserData(user)
+            } else {
+                avatarView.isVisible = false
+            }
+        }
+
+        with(threadsFootnote) {
+
+            root.updateConstraints {
+                clear(threadRepliesButton.id, ConstraintSet.TOP)
+                connect(
+                    threadRepliesButton.id,
+                    ConstraintSet.TOP,
+                    if (isMine) firstMineUserImage.id else firstTheirUserImage.id,
+                    ConstraintSet.TOP
+                )
+            }
```

Comment:
Why do we have to update this vertical constraint every time? Couldn't we constrain this View in a fixed way instead?

Reply:
Thanks, I found out how to achieve it